VoiceControl/VoiceControl.cpp

```diff
@@ -40,13 +40,15 @@ namespace WPEFramework {
             LOGINFO("ctor");
             VoiceControl::_instance = this;
 
-            Register("getApiVersionNumber",   &VoiceControl::getApiVersionNumber, this);
+            Register("getApiVersionNumber",   &VoiceControl::getApiVersionNumber,   this);
 
-            Register("voiceStatus",           &VoiceControl::voiceStatus,         this);
-            Register("configureVoice",        &VoiceControl::configureVoice,      this);
-            Register("setVoiceInit",          &VoiceControl::setVoiceInit,        this);
-            Register("sendVoiceMessage",      &VoiceControl::sendVoiceMessage,    this);
-            Register("voiceSessionByText",    &VoiceControl::voiceSessionByText,  this);
+            Register("voiceStatus",           &VoiceControl::voiceStatus,           this);
+            Register("configureVoice",        &VoiceControl::configureVoice,        this);
+            Register("setVoiceInit",          &VoiceControl::setVoiceInit,          this);
+            Register("sendVoiceMessage",      &VoiceControl::sendVoiceMessage,      this);
+            Register("voiceSessionTypes",     &VoiceControl::voiceSessionTypes,     this);
```

Comment:
Pls update the VoiceControl.json schema to reflect the deprecated API and also add the new APIs and regenerate VoiceControlPlugin.md.